lib/routes/account.js

```diff
@@ -650,12 +650,14 @@ module.exports = function (
               // If the account is already verified, they may be e.g.
               // clicking a stale link.  Silently succeed.
               if (account.emailVerified) {
+                log.activityEvent('account.already_verified', account.uid, request)
```

Comment:
Thinking about these "activity events", I don't think it's useful to add any more until we settle on a path forward.  "re-verified an existing account" is not something we'd track on a per-user basis for metrics purposes.

If we're interested in counting this as a stat, it may be clearer to add a `log.increment` function or similar that just directly does what we want without any implied higher purpose.